src/lib/transformers.js

```diff
@@ -0,0 +1,40 @@
+const colors = require('colors');
+
+/* istanbul ignore next */
+const padd = (v) => ` ${v} `;
+
+/* istanbul ignore next */
+module.exports.defaultTransformers = {
+    cl_red: s => colors.red(s),
+    cl_blue: s => colors.blue(s),
+    cl_cyan: s => colors.cyan(s),
+    cl_green: s => colors.green(s),
+    cl_gray: s => colors.gray(s),
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
    cl_gray: (s) => colors.gray(s),
```